python/cuml/fil/README.md

```diff
@@ -0,0 +1,74 @@
+# FIL - RAPIDS Forest Inference Library
+
+The Forest Inference Library provides a lightweight, flexible API to
+infer (predict) results from a tree-based model ensemble on GPU. The
+tree ensemble can be either gradient-boosted decision tree (GBDT) or
+random forest (RF) models trained in XGBoost, cuML, scikit-learn, or
+LightGBM.
+
+# Code sample
+
+Starting with an XGBoost classification model saved in the file
+"xgb.mod," we want to use that model to infer on a large dataset of
+test samples.
+
+```python
+from cuml import ForestInference
+
+fm = ForestInference.load(filename=model_path,
+                          output_class=True,
+                          threshold=0.50,
+                          model_type='xgboost')
+
+X = ... load test samples as a numpy or cupy array ...
+
+y_out = fm.predict(X)
+
+```
+
```

Comment:
Do we talk about the C API? Given that we're currently testing and maintaining it as well as the python API.

Reply:
Yeah, good point... we don't have a really nice sample of forest inference from C yet right? I propose we mention the C API and include a link to the docs but don't emphasize it until we have a nice clean example to share.

Reply:
Our API docs don't really make it clear which bits are user-facing vs. internal: https://docs.rapids.ai/api/libcuml/nightly/namespaceML_1_1fil.html
Maybe there is a doxygen trick we could do to hide `node_*` in those docs?